src/rpc/mining.cpp

```diff
@@ -489,7 +489,7 @@ static RPCHelpMan getprioritisedtransactions()
         RPCResult{
             RPCResult::Type::OBJ_DYN, "", "prioritisation keyed by txid",
             {
-                {RPCResult::Type::OBJ, "txid", "", {
+                {RPCResult::Type::OBJ, "transactionid", "", {
```

Comment:
updated the commit message in [9a918a4](https://github.com/bitcoin/bitcoin/pull/28885/commits/9a918a4720ee091344a16dcd703d169ebdb7938e) 

I also fixed the other ones that had refactor in the commit message since they weren't really refactors either